dist/686C-674-schema.json

```diff
@@ -0,0 +1,1645 @@
+          "$ref": "#/definitions/genericLocation"
+        },
+        "isMarriageAnnulledOrVoid": {
+          "$ref": "#/definitions/genericTrueFalse"
+        },
+        "explanationOfAnnullmentOrVoid": {
+          "type": "string",
+          "maxLength": 500,
+          "pattern": "^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*]+$"
+        }
+      }
+    },
+    "deceasedDependents": {
+      "type": "object",
+      "properties": {
+        "dependentInformation": {
+          "type": "object",
+          "properties": {
+            "deaths": {
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "type": "object",
+                "properties": {
+                  "dependentType": {
+                    "type": "string",
+                    "enum": [
+                      "SPOUSE",
+                      "DEPENDENT_PARENT",
+                      "CHILD"
+                    ],
+                    "enumNames": [
+                      "Spouse",
+                      "Dependent Parent",
+                      "Child"
+                    ]
+                  },
+                  "childStatus": {
+                    "type": "object",
+                    "properties": {
+                      "childUnder18": {
+                        "$ref": "#/definitions/genericTrueFalse"
+                      },
+                      "stepChild": {
+                        "$ref": "#/definitions/genericTrueFalse"
+                      },
+                      "adopted": {
+                        "$ref": "#/definitions/genericTrueFalse"
+                      },
+                      "disabled": {
+                        "$ref": "#/definitions/genericTrueFalse"
+                      },
+                      "childOver18InSchool": {
+                        "$ref": "#/definitions/genericTrueFalse"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "dependentAdditionalInformation": {
+        "type": "object",
+        "properties": {
+          "deaths": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "type": "object",
+              "properties": {
+                "deceasedDateOfDeath": {
+                  "$ref": "#/definitions/date"
+                },
+                "deceasedLocationOfDeath": {
+                  "$ref": "#/definitions/genericLocation"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "reportChildMarriage": {
+      "type": "object",
+      "properties": {
+        "dateChildMarried": {
+          "$ref": "#/definitions/date"
+        }
+      }
+    },
+    "reportChildStoppedAttendingSchool": {
+      "type": "object",
+      "properties": {
+        "dateChildLeftSchool": {
+          "$ref": "#/definitions/date"
+        }
+      }
+    },
```

Comment:
Looks like all the fields that use the imported `fullName` property need to be updated because that's getting left out of the built schema in dist.

Reply:
So I actually decided to revert this to using our definition of fullName since we are doing a separate ticket with importing the common definitions